### DISCUSSION TITLE
Showing score while playing

### DISCUSSION THREAD

**Discussion by V3rald** [ppy/osu#27937] [Q&A] [answered]
Is there any way to see the scores, combo etc while playing on a map? (the leaderboard i mean)
I can only see the rank and avatar.

**Comment by Joehuu** [comment, marked as answer]
Not yet, unfortunately. There have been attempts to bring this setting but had the wrong direction: https://github.com/ppy/osu/pull/24947, https://github.com/ppy/osu/pull/27285

It's planned to be another skinnable component for the skin editor, so more settings can be easily added.
